### PR TITLE
IOS/ES: Make NandUtils member functions

### PR DESCRIPTION
**Pull request by leoetlino**
In the future, NAND filesystem access will be limited to one IOS instance, for safety reasons and to make it possible to consider supporting NAND images. This means that any code accessing the NAND filesystem must go through the FS device, both for code that is external to IOS and internal.

Because we don't want to introduce any singleton, this requires internal IOS code that needs NAND access to be part of an IOS device class, so they can access the FS device easily.

Making some of the internal ES implementation functions member functions also prevents them from being (mis)used outside of IOS, since they cannot be called everywhere anymore.

### CODE REVIEW

File: Source/Core/Core/IOS/ES/ES.h [hunk 1]
```diff
@@ -284,6 +298,16 @@ class ES final : public Device
   bool LaunchPPCTitle(u64 title_id, bool skip_reload);
   static TitleContext& GetTitleContext();
 
+  // Start a title import.
+  bool InitImport(u64 title_id);
```
Comment (iwubcode) [inline Source/Core/Core/IOS/ES/ES.h]: Should any of these be private?  If the goal is to not misuse them then it feels like their open access is not ideal.

Comment (leoetlino) [inline Source/Core/Core/IOS/ES/ES.h]: These are private

Comment (iwubcode) [inline Source/Core/Core/IOS/ES/ES.h]: Ah, so sorry, I see that now.  I must have missed it last night...

Comment (leoetlino) [inline Source/Core/Core/IOS/ES/ES.h]: No worries. GitHub just tends to hide the important lines :P

**Pull request merged**